src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -259,13 +260,21 @@ public void getUrl(String url, boolean makeScreenshot) {
     }
   }
 
+  public void getUrl(Page page) {
+    getUrl(urlBuilder.getUrlForPage(page));
+  }
+
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)